install.sh

```diff
@@ -15,10 +15,14 @@ TARGET=${TARGET-linux}
 libversion=3
 libVFile="./libversion"
 
+function has_cmd {
+	command -v "$1" >/dev/null 2>&1
```

Comment:
I don't like that we blindly assume that `command -v` is harmless.

But with your link it's OK to leave it as is for our limited usage.